udata_front/theme/gouvfr/assets/js/components/utils/read-more.vue

```diff
@@ -11,8 +11,10 @@ A very simple component that display its content up to a maximum height and then
 -->
 
 <template>
-  <div ref="container" class="read-more" :class="{expand: expanded}" :style="{height: containerHeight + 'px'}">
-    <slot></slot>
+  <div class="relative">
+    <div ref="container" class="read-more" :class="{expand: expanded}" :style="{height: containerHeight + 'px'}">
```

Comment:
I think you may need to change the height of the shading, it cuts a bit too soon when reading more.
Ex [on the SIREN](http://dev.local:7000/fr/datasets/base-sirene-des-entreprises-et-de-leurs-etablissements-siren-siret/)
![image](https://github.com/etalab/udata-front/assets/28541902/a2954ad3-e3cc-4bf1-a6e0-4ca3535f4e9e)

Reply:
I forgot to update the CSS after the HTML changes. Done.